### ISSUE TITLE
[Bug] error while loading shared libraries: libevent_core-2.1.so.7

### ISSUE THREAD

**Issue by KaKi87** [casperklein/docker-tmux-builder#1]
Hello,

I'm experiencing the following error after installing [`tmux_3.5a-1_amd64.deb`](https://github.com/casperklein/docker-tmux-builder/releases/download/3.5a/tmux_3.5a-1_amd64.deb) on Debian 12.8 :

```
tmux: error while loading shared libraries: libevent_core-2.1.so.7: cannot open shared object file: No such file or directory
```

Thanks

**Comment by casperklein** [comment]
You have to install the dependency: 

```bash
apt update
apt install libevent-core-2.1-7
```

**Comment by KaKi87** [comment]
That works, thanks.
So, I would like to suggest declaring that dependency in the DEB.
Thanks

**Comment by casperklein** [comment]
If you use `make install`, `libevent-core` will be installed automatically:

https://github.com/casperklein/docker-tmux-builder/blob/43ff757a0d596340fcda146a3a3ee9b7738333ed/Makefile#L27

I haven't add `libevent-core` as dependency to make it easier, to build `tmux` for older debian versions, e.g. the package name is different in Debian 10 (`libevent-core-2.1-6`).

TIL that you can use `apt install ./tmux_3.5a-1_amd64.deb`, that will automatically install dependencies. I always used `dpkg -i ./tmux_3.5a-1_amd64.deb`  which fails, if `libevent-core` isn't installed manually beforehand.

I don't think there is great demand for backward compatibility + the `Dockerfile` would still require other adjustments to make it work. I've added the dependency: b8c8f4d7acded648d1d8df4fa08f68b4da9a9ba6

**Comment by KaKi87** [comment]
> TIL that you can use `apt install ./tmux_3.5a-1_amd64.deb`, that will automatically install dependencies.

That's precisely what I was talking about, except the metadata of the DEB you built doesn't declare this dependency.

Here's the output of `dpkg -I tmux_3.5a-1_amd64.deb` (metadata of your DEB) :

```
 new Debian package, version 2.0.
 size 459172 bytes: control archive=440 bytes.
       0 bytes,     0 lines      conffiles            
     301 bytes,     9 lines      control              
 Package: tmux
 Priority: extra
 Section: admin
 Installed-Size: 1224
 Maintainer: casperklein@docker-tmux-builder
 Architecture: amd64
 Version: 3.5a-1
 Provides: tmux
 Description: tmux is a terminal multiplexer: it enables a number of terminals to be created, accessed, and controlled from a single screen.
```

While here's the output of `dpkg -I tmux_3.3a-3_amd64.deb` (metadata of the official older Debian DEB) :

```
 new Debian package, version 2.0.
 size 454536 bytes: control archive=1380 bytes.
    1024 bytes,    21 lines      control              
     511 bytes,     8 lines      md5sums              
      99 bytes,     9 lines   *  postinst             #!/bin/sh
      84 bytes,     9 lines   *  postrm               #!/bin/sh
     228 bytes,    10 lines   *  preinst              #!/bin/sh
 Package: tmux
 Version: 3.3a-3
 Architecture: amd64
 Maintainer: Romain Francoise <rfrancoise@debian.org>
 Installed-Size: 1106
 Depends: libc6 (>= 2.34), libevent-core-2.1-7 (>= 2.1.8-stable), libtinfo6 (>= 6), libutempter0 (>= 1.1.5)
 Section: admin
 Priority: optional
 Homepage: https://tmux.github.io/
 Description: terminal multiplexer
  tmux enables a number of terminals (or windows) to be accessed and
  controlled from a single terminal like screen. tmux runs as a
  server-client system. A server is created automatically when necessary
  and holds a number of sessions, each of which may have a number of
  windows linked to it. Any number of clients may connect to a session,
  or the server may be controlled by issuing commands with tmux.
  Communication takes place through a socket, by default placed in /tmp.
  Moreover tmux provides a consistent and well-documented command
  interface, with the same syntax whether used interactively, as a key
  binding, or from the shell. It offers a choice of vim or Emacs key
  layouts.
```

See the specific relevant line :

```
Depends: libc6 (>= 2.34), libevent-core-2.1-7 (>= 2.1.8-stable), libtinfo6 (>= 6), libutempter0 (>= 1.1.5)
```

Thanks

**Comment by casperklein** [comment]
> That's precisely what I was talking about, except the metadata of the DEB you built doesn't declare this dependency.

That has been fixed with the above mentioned commit b8c8f4d7acded648d1d8df4fa08f68b4da9a9ba6 + a17f5c0ba0b790a81cfc1488c0f42c2df240ad6b
